demo_docker/docker-compose.yaml

```
@@ -73,6 +78,7 @@ services:
       - '-c'
       - |
         # NOTE: firm_demo0_eimとfirm_demo1_eimの実行完了を確認するまで実行しない
+        if [ -f /sync/firm_demo1_eim_failed ]; then exit 1; fi && \
         while [ ! -f /sync/firm_demo0_eim_done ] || [ ! -f /sync/firm_demo1_eim_done ]; do sleep 1; done && \
```

`/sync/firm_demo1_eim_failed` の存在検出が最初に一回しか行われない場合、matching よりも先にこちらが処理される場合に検出に失敗しそうです。
例えば以下のように出来るかなと思いますが、シェルは不慣れなのでもっといい書き方があればそちらを採用してください。
```suggestion
        while [ ! -f /sync/firm_demo0_eim_done ] || [ ! -f /sync/firm_demo1_eim_done ]; do \
            if [ -f /sync/firm_demo1_eim_failed ]; then exit 1; fi && \
            sleep 1; \
        done && \
```

---

こちら良さそうだったのでコミットしています！